architecture/additional_concepts/flannel.adoc

```diff
@@ -0,0 +1,32 @@
+[[architecture-additional-concepts-flannel]]
+= Flannel
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[architecture-additional-concepts-flannel-overview]]
+== Overview
+
+*flannel* is a virtual networking layer designed specifically for containers. 
+{product-title} can use it for networking containers instead of the default
+software-defined networking (SDN) components.
+
+[[architecture-additional-concepts-flannel-architecture]]
+== Architecture
+
+Each host within the network runs an agent called *flanneld*, which is
+responsibile for:
+
+- Managing a unique subnet on each host
+- Distributing IP addresses to each container on its host
+- Mapping routes from one container to another, even if on different hosts
+
+Each *flanneld* agent provides this infomation to a centralized *etcd* store so
+other agents on hosts can create an overlay network and route packets to
+any container within the *flannel* network.
```

Comment:
Probably a good idea to do so. I mention why in install_config/configuring_sdn.adoc, but it might make sense to add it here too.